pkgs/top-level/all-packages.nix

```diff
@@ -35620,4 +35620,8 @@ with pkgs;
   honeyvent = callPackage ../servers/tracing/honeycomb/honeyvent { };
 
   mictray = callPackage ../tools/audio/mictray { };
+
+  vpype = with python3Packages; toPythonApplication (callPackage ../applications/graphics/vpype { });
```

Comment:
generally `toPythonApplication` is to take python packages and make them into venv-like applications. But it's odd that you're taking something from a non-python module path